components/telemetry-operator/main.go

```diff
@@ -118,6 +119,7 @@ func main() {
 	flag.StringVar(&supportedOutputPlugins, "supported-output-plugins", "", "Comma separated list of supported output plugins. If empty, all output plugins are allowed.")
 	flag.StringVar(&deniedFilterPlugins, "denied-filter-plugins", "", "Comma separated list of denied filter plugins even if allowUnsupportedPlugins is enabled. If empty, all filter plugins are allowed.")
 	flag.StringVar(&deniedOutputPlugins, "denied-output-plugins", "", "Comma separated list of denied output plugins even if allowUnsupportedPlugins is enabled. If empty, all output plugins are allowed.")
+	flag.IntVar(&maxPipelines, "max-pipelines", 5, "Maximum number of log pipelines to be created. If 0, no limit is applied.")
```

Comment:
> If 0, no limit is applied

Is this covered by the current application?

Reply:
Good catch, covered that case in the validation.